src/main/kotlin/study/racingCar/view/ResultView.kt

```diff
@@ -1,14 +1,18 @@
 package study.racingCar.view
 
 object ResultView {
-    fun printResult(movedStepsRecords: List<MutableList<Int>>) {
-
-        movedStepsRecords.forEach { eachTryRecords ->
-            eachTryRecords.forEach { movedSteps ->
-                println("-".repeat(movedSteps))
-                println()
+    fun printResult(mileageRecords: List<List<Pair<String, Int>>>) {
+        mileageRecords.forEach { eachTryRecords ->
+            eachTryRecords.forEach { pair ->
+                println("${pair.first} :  " + "-".repeat(pair.second))
             }
```

Comment:
`Pair`는  다음과 같이 이름을 줄 수도 있어요! 그리고 `String Template`도 잘 사용해주셨네요! 다음과 같이도 활용할 수 있지 않을까요?
```suggestion
            eachTryRecords.forEach { (name, mileage) ->
                println("$name : ${"-".repeat(mileage)}")
            }
```